core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +44,28 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Get the event types that this class suscribe to.
```

Comment:
Insert new line here.
```Python
"""Brief description about the function.

    Args:
        arg1: type. Short description.
        arg2: type. Short description.

    Returns:
        type. Short description.

    Raises:
        TypeOfException: Short description.
    """
```

Reply:
So sorry for not catching this earlier but there is a typo and grammar mistake here :
`...that this class suscribe to.` ->  `...that this class subscribes to.` (ditto below)